### ISSUE TITLE
Issue on Rakefile

### ISSUE THREAD

**Issue by esttorhe** [ashfurrow/blog#224]
Don't know if its a real issue or not but I was checking your `Rakefile` and trying to improve upon my own for my blog and came to this line that was causing a crash https://github.com/ashfurrow/blog/blob/master/Rakefile#L160

After trying and talking with some `Ruby` devs y changed the `require` to something like this:
```ruby
require 'html-proofer'
```

And it worked.

Don't know if its something specific to my configuration or not but thought it might be worth double checking just in case ¯\\\_(ツ)\_/¯

---

# TL;DR

Change the `/` to `-` in the `require` here https://github.com/ashfurrow/blog/blob/master/Rakefile#L160

**Comment by esttorhe** [comment]
Just realized the `TL;DR` might come in as «harsh».

So I decided to add a clarification.

Shouldn't read ~Change~ and instead should be
>Should `/` be changed for `-`

I'm no `Ruby` dev its more a question 🙈

**Comment by ashfurrow** [comment]
No problem about the tone – always okay to clarify though 👍

I think you're right about this fix, which leaves me curious about how CI has been actually working this whole time. Maybe the gem just happened to be loaded into the Ruby runtime already, the `require` returned false but didn't raise an exception, and the code executed as expected.

**Comment by ashfurrow** [comment]
Ah, even more confusing! So html-proofer is currently on 3.x with breaking changes from the version _I'm_ using, 2.x. The readme for the version of html-proofer [locked](https://github.com/ashfurrow/blog/blob/master/Gemfile.lock#L215) in my `Gemfile.lock` [says](https://github.com/gjtorikian/html-proofer/tree/002c5f1a578f045ff52f53e447aee16889db2b7a) to `require 'html/proofer'`, even though the current one is `require 'html-proofer'`. So it looks thing things work for my outdated codebase but another dev referring to my Gemfile and Rakefile would install the latest version and it wouldn't work for them.

Esteban, if you're in this situation, I reckon you could either update the Rakefile code to use the new html-proofer version, or lock the gem to the older version in your Gemfile: `gem 'html-proofer', '< 3'`. 3.0 has only been out since March, so I'm just going to stick with 2.x for now.

**Comment by esttorhe** [comment]
Yeah I noticed the breaking changes.

I actually went ahead and updated the `Rakefile` to the new syntax and module names, also changed the `require` sentence and everything is working now but thought it was worth raising the issue for you to check.

Glad you figured it out 👍

**Comment by ashfurrow** [comment]
💯🙇
